judge/views/submission.py

```diff
@@ -576,11 +577,11 @@ def _get_result_data(self, queryset=None):
 
 
 class ForceContestMixin(object):
-    @property
+    @cached_property
     def in_contest(self):
         return True
 
-    @property
+    @cached_property
```

Comment:
what does this change do?
does mypy complain less with `@cached_property`?

Reply:
These two properties override the ones in `SubmissionsListBase`, which are `cached_property`s.